config/config.go

```diff
@@ -71,18 +73,22 @@ type Loader struct {
 }
 
 // DefaultLoader is going to be used by a service if config is not specified.
-var DefaultLoader = NewLoader()
+// First values are going to be looked in dynamic providers, then in command line provider
+// and YAML provider is going to be the last.
+var DefaultLoader = NewLoader(commandLineProviderFunc)
```

Comment:
Should we wrap `commandLineProviderFunc` inside NewLoader as default rather than passing it in?